docs/modules/operation/pages/provisioning/detectors/Win32ServiceDetector.adoc

```diff
@@ -1,6 +1,7 @@
 = Win32 Service Detector
 
-The Win32 Service Detector can be used for detecting the availability of services on Windows devices with SNMP enabled. It will make a query against the Microsoft LAN Manager MIB `.1.3.6.1.4.1.77` over SNMP to confirm the service.
+The Win32 Service Detector can be used for detecting the availability of services on Windows devices with SNMP enabled.
+It will make a query against the Microsoft LAN Manager MIB `.1.3.6.1.4.1.77` over SNMP to confirm the service.
```

Comment:
It will query against the Microsoft LAN Manager MIB `.1.3.6.1.4.1.77` over SNMP to confirm the service.